src/cargo/ops/tree/mod.rs

```diff
@@ -341,15 +369,15 @@ fn print_node<'a>(
 
 /// Prints all the dependencies of a package for the given dependency kind.
 fn print_dependencies<'a>(
-    gctx: &GlobalContext,
+    ws: &Workspace<'_>,
```

Comment:
nit: splitting the change in parameter would shrink the focus of the main commit to make it easier to see whats happening